### PR TITLE
neofetch: add version 7.1.0

### PR DESCRIPTION
**Pull request by torculus**
Adding `neofetch`, a bash utility for displaying system information

### CODE REVIEW

File: components/sysutils/neofetch/manifests/sample-manifest.p5m [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2021 Benjamin S. Osenbach
+#
```
Comment (AndWac) [inline components/sysutils/neofetch/manifests/sample-manifest.p5m:14]: You have mixed the places for the Copyright. The sample-manifst should stay as it is generated (it's both, a template for the real manifest file and a baseline to check if something has changed in a new version).

File: components/sysutils/neofetch/neofetch.p5m [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2021 <contributor>
+#
```
Comment (AndWac) [inline components/sysutils/neofetch/neofetch.p5m:14]: Your Copyright belongs here.

Comment (torculus) [reply components/sysutils/neofetch/neofetch.p5m:14]: Okay, I just went ahead and added a new commit to fix it

**Comment by pyhalov** [comment]
It shows:
```
Packages: 354 (pkginfo), 1 (pkg)
```
Likely it needs the following fix:
```
--- neofetch.1  2021-05-26 21:22:22.828639039 +0000
+++ neofetch    2021-05-26 21:22:10.350187007 +0000
@@ -1585,6 +1585,9 @@
             case $kernel_name in
                 FreeBSD|DragonFly) has pkg && tot pkg info ;;
 
+               SunOS)
+                       has pkg && tot pkg list ;;
+
                 *)
                     has pkg && dir /var/db/pkg/*
 

```

**Comment by torculus** [comment]
Okay, I have patched the $kernel_info and can confirm that pkg is now showing more installed packages

File: components/sysutils/neofetch/patches/01-fix-pkginfo.patch [hunk 3]
```diff
@@ -0,0 +1,11 @@
+--- neofetch.1  2021-05-26 21:22:22.828639039
++++ neofetch    2021-05-26 21:22:10.350187007
+@@ -1585,6 +1585,9 @@
```
Comment (AndWac) [inline components/sysutils/neofetch/patches/01-fix-pkginfo.patch]: Your Makefile settings and the patch file don't match.
You either need to create a patch file with paths (which I recommend) or
add PATCH_LEVEL= 0 to your Makefile.
Another thing: your copy's name of neofetch.1 with the unmodified original contents clashes with the man file that is part of the sources.
My recommendation is here to use another name. My usual workflow would be like this:
cd neofetch-7.1
cp -p neofetch neofetch.orig
vi neofetch   (make the changes)
cd ..
gdiff -u neofetch-7.1/neofetch.orig neofetch-7.1/neofetch >patches/01-fix-pkginfo.patch
(Using gdiff is not necessary; requirement is just a unified diff)

Comment (torculus) [reply components/sysutils/neofetch/patches/01-fix-pkginfo.patch]: Thanks, I've rebased and updated the patch file

**Comment by AndWac** [comment]
Something went wrong with your rebasing. You got a commit into your PR that belongs to the last merged commit.
And while we are at rebasing: the last step before merging a PR would be to squash your commits into a single one leaving only one appropriate commit message.

**Comment by torculus** [comment]
Okay, I have rebased (again), squashed, and force-pushed the commits to my repo. Please let me know if there's anything else needed

**Review by AndWac — APPROVED** [review]

**Comment by AndWac** [comment]
Ok, just one more thing left: Please also squash the commit messages. When you rebased you kept all the commit messages. Just keep one with an appropriate text.

**Comment by torculus** [comment]
> Ok, just one more thing left: Please also squash the commit messages. When you rebased you kept all the commit messages. Just keep one with an appropriate text.

Okay, I just squashed the commit messages too. Thank you for all of your help!

**Comment by AndWac** [comment]
Ok, now please rethink your commit message :)
That you have squashed commits is not relevant for the commit and the PR.

**Comment by torculus** [comment]
> Ok, now please rethink your commit message :)
> That you have squashed commits is not relevant for the commit and the PR.

I based the format of the commit message on approved pull requests

**Comment by AndWac** [comment]
Ok, now I ruined your commit message :/

Thank you for your contribution!

**Comment by torculus** [comment]
That's okay, I'm just glad it went through :)
Again, thank you for your help!

**Comment by AndWac** [comment]
Am 28.05.21 um 22:22 schrieb Benjamin S. Osenbach:
>
> That's okay, I'm just glad it went through :)
> Again, thank you for your help!
>
>
You are welcome, but I think Alexander did the most important part.
And I hope for more contributions from you in the future ;)

**Pull request merged**